pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -54,11 +54,19 @@ type BuildSpec struct {
 	CacheName      string                      `json:"cacheName"`
 	Env            []corev1.EnvVar             `json:"env"`
```

Comment:
What do y'll think about adding the RunImage to the spec instead of inside the RebaseConfig?

Reply:
The only reason I am on the fence about this is because one the build is built, it will involve just duplicating the exact same info from the spec into the status. Maybe that isnt that big a deal, but it feels weird.